src/JuMP.jl

```diff
@@ -395,6 +396,36 @@ function add_bridge(model::Model,
     return
 end
 
+function _moi_add_bridge(model::Nothing,
+                        BridgeType::Type{<:MOI.Bridges.AbstractBridge})
+    # No optimizer is attached, the bridge graph will be printed if one is attached
+    return
+end
```

Comment:
What is this function for?

Reply:
Honestly speaking I am also not very sure abt that, I added it because the same kind of the function is also there for adding bridges (maybe it is there to not get an error if we try to add bridge in empty Model)  
`function _moi_add_bridge(model::Nothing,
                        BridgeType::Type{<:MOI.Bridges.AbstractBridge})
    # No optimizer is attached, the bridge will be added when one is attached
    return
end`

Reply:
Please remove.

Reply:
cool.